enhancements/78-dynamically-reclaiming-resources/README.md

```diff
@@ -0,0 +1,142 @@
+# Dynamically reclaim resources of Pods of a Workload
```

Comment:
We added a template now https://github.com/kubernetes-sigs/kueue/tree/main/keps/NNNN-template

Please move the file to keps/

Reply:
Moved to keps/ directory.